src/router/index.js

```diff
@@ -14,7 +14,7 @@ const routes = [
     component: Lists,
   },
   {
-    path: '/Lists/:title',
+    path: '/Lists/:listId',
```

Comment:
having the list title in the URL is going to be difficult when we have two lists with the same name.